### DISCUSSION TITLE
How to use the new version?

### DISCUSSION THREAD

**Discussion by b0iq** [infinitered/ignite#2086] [Q&A] [answered]
I was wondering if I can try the next version, are there any steps that can I follow to start testing it?

I know you guys are working so hard to release the beta version within the next days but I think if you provide a way to install the under development version; that is going to let other developers helping with and prevent future issues from happening.

THANK YOU ALL.

Sorry for my English :p

**Comment by jamonholmgren** [comment, marked as answer]
Hey @b0iq ! Thanks for your interest in Ignite v8: Maverick!

To test the new version, do this:

```
# just in case
npm uninstall -g ignite-cli

git clone https://github.com/infinitered/ignite.git
cd ignite
git checkout maverick
yarn
yarn link
```

Then, in a temp folder:

```
ignite new PizzaApp
```

This will use the new Maverick version!

Anytime you want to update to the latest:

```
cd ignite
git pull
yarn
```

... and continue using it.

When you're done testing:

```
cd ignite
yarn unlink
```

You don't want that alpha version hanging around.

**Reply by jamonholmgren** [reply]
Additionally, you can run the boilerplate directly from Maverick:

```
cd ignite/boilerplate
yarn
yarn ios
yarn android
yarn expo:start
```
